Import from hub docs (#5631)
recognai/rubrix: argilla/docs/getting_started/quickstart.md

Argilla will automatically interpret the columns in the dataset to map them to Fields and Questions.

![Screenshot of the dataset configuration page](...)

**Fields** include the data that you want feedback on, like text, chats, or images. If you want to exclude any of the Fields that Argilla identified for you, simply select the "No mapping" option.

**Questions** are the feedback you want to collect, like labels, ratings, rankings, or text. If Argilla identified questions in your dataset that you don't want, you can eliminate them. You can also add questions of your own.

!!! note
    You will be able to modify some elements of the configuration of the dataset after it has been created from the Dataset Settings page e.g., the titles of fields and questions. Check all the settings you can modify in the [Update a dataset](../how_to_guides/dataset.md#update-a-dataset) section.

When you're happy with the result, you'll need to give a name to your dataset, select a workspace and choose a split, if applicable. Then, Argilla will start importing the dataset in the background. Now you're all set up to start annotating!

!!! note

>     If your dataset is bigger than 10k records, at this stage Argilla will only import the first 10k. You can import the rest of the records at any point using the Python SDK.

Reviewer: ```suggestion
    Argilla will only import the first 10k rows of a dataset. If your dataset is larger, you can import the rest of the records at any point using the Python SDK.
```

```

> ## Create your first dataset

Reviewer: We could actually just lose this section, and reference the guide on creating datasets in the SDK. 🤔

🎉 You have successfully created your first dataset with Argilla. You can now access it in the Argilla UI and start annotating the records.

## Next steps

> 

Reviewer: We could have a next step to upload data to the hub, so that users could use this feature without code? Or maybe this should be higher up. 🤔
